build/fuse-clients/ossfs/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM registry.cn-hangzhou.aliyuncs.com/acs/alinux:3-update
```

Comment:
We are using `registry.eu-west-1.aliyuncs.com/acs/alinux:3-update` in multi/Dockerfile.multi, should we be consistent here?

Reply:
image localcache could be garbage collected, IMO, image manifest should be kept as small as possible